lib/smart_proxy_container_gateway/container_gateway.rb

```diff
@@ -41,7 +41,7 @@ class Plugin < ::Proxy::Plugin
         container_instance.singleton_dependency :container_gateway_main_impl, (lambda do
           Proxy::ContainerGateway::ContainerGatewayMain.new(
             database: container_instance.get_dependency(:database_impl),
-            **settings.slice(:pulp_endpoint, :pulp_client_ssl_ca, :pulp_client_ssl_cert, :pulp_client_ssl_key)
+            **settings.slice(:pulp_endpoint, :pulp_client_ssl_ca, :pulp_client_ssl_cert, :pulp_client_ssl_key, :loadbalancer)
```

Comment:
Since the config option naming came up:
```suggestion
            **settings.slice(:pulp_endpoint, :pulp_client_ssl_ca, :pulp_client_ssl_cert, :pulp_client_ssl_key, :client_content_endpoint)
```
This configurable option determines where we redirect the client to to get their requested content. So, I propose that the generic name be `client_content_endpoint`.